app/assets/javascripts/modules/media_viewer.js

```diff
@@ -146,11 +147,22 @@
       return hlsSource.prop('src').indexOf('/mp3:') >= 0;
     }
 
+    function ms10Browser() {
+      var fingerprinter = new ClientJS();
```

Comment:
Adding `/*global ClientJS */` to the top of the file is how you will make Hound quiet (which is what we're doing there w/ the `ThumbSlider` and `videojs` globals).